docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -510,24 +510,26 @@ Post-Release Review Process
 
      XXX: This step should be automated. See `FLOC-1039 <https://clusterhq.atlassian.net/browse/FLOC-1039>`_.
 
-   * The node package (``flocker-node``) should be installed on all supported platforms.
-     You can use vagrant to boot a clean Fedora 20 machine as follows:
+   * The node package (``flocker-node``) should be installed and tested on all supported platforms.
+
+     The acceptance tests can be run using the ``run-acceptance-tests`` tool.
```

Comment:
Note that there is a race condition here. If https://clusterhq.atlassian.net/browse/FLOC-1304 gets completed before this is merged, this will add unnecessary steps.